src/system/Footer/Footer.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Box } from '../Box/Box';
+import { Flex } from '../Flex';
+import { LinkExternal, LinkExternalProps } from '../LinkExternal/LinkExternal';
+import { Text } from '../Text';
+
+type FooterProps = {
+	/**
+	 * Add an additional separator after the last element.
+	 *
+	 * @default false
+	 **/
+	hasTrailingSeparator?: boolean;
+	/**
+	 * An array of LinkExternal components as objects.
+	 **/
+	links: LinkExternalProps[];
+	/**
+	 * A logo to display in the right footer area. Displays Automattic logo by default.
+	 **/
+	customLogo?: React.ReactNode;
+	/**
+	 * The maxiumum width of the footer.
+	 *
+	 * @default 100%
+	 **/
+	maxWidth?: string | number;
+};
+
+const a8cLogo = (
```

Comment:
Maybe a different file would fit better for this?

Reply:
Yes, that would make sense. I initially debated on having it as a default at all so before it was only in the story example, but then I thought it would be good for consistency. And if it reused anywhere else, would definitely be cleaner in another file. 

Do you think in the `Footer` folder or maybe in an `assets` folder?

Reply:
I went with adding an assets folder: https://github.com/Automattic/vip-design-system/pull/409/commits/3cb2ad656f115a5baf2739d50f8fc8c03f37598a